instance/api/instance.py

```diff
@@ -72,16 +72,22 @@ def list(self, request):
         List all instances. No App server list is returned in the list view, only the newest app server information.
 
         """
+        # Don't load all columns, because some of them have very big data
+        appservers_few_columns = OpenEdXAppServer.objects.only('_is_active', '_status', 'id', 'name', 'owner_id',
+                                                               'created', 'modified')
         queryset = self.queryset.prefetch_related(
             # Use prefetching to make the number of database queries required to
             # generate this list O(1).
-            # Note that prefetching all app server information is still required, as the "newest" is not decideable
+            # Note that prefetching all app servers information is still required, as the "newest" is not decideable
```

Comment:
The older comment was actually grammatically correct. :p

Reply:
It's not about grammar, it's about the word „all“. Before the patch, it was really loading all „server information“. Now, it's not loading all information, just a part of it. But it's still true that it's loading information for all app servers…
I can change it to something else.

Reply:
Nevermind, it's not that important :)